sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,5 +1,5 @@
 # Release History
-## 5.1.0-beta.1 (Unreleased)
+## 5.1.0-beta.1 (09/17/2020)
```

Comment:
version_client.txt changes?

Reply:
talked offline. After released 5.0.0 there is an automatically PR to increase version 5.1.0-beta.1. So the version_client.txt is already merged in that PR.